What-2-Watch/what-2-watch-fe
# Feature/user services

## Is this a fix or a feature?
Feature
## What is the change?
Adds service functionality to the user creation and user profile page. 
## What does it fix?
Changes storage of userID from data in app to local storage
## Where should the reviewer start?
Review the services component
## How should this be tested?
Run the dev server, confirm that selected services from user creation are present within the user profile.
